mission/bylaws.md

```diff
@@ -0,0 +1,14 @@
+#Draft founding principles for the Foundation for Public Code
+
+## Mission
+To grow an ecosystem of public purpose open source policy and software.
```

Comment:
This bears some further work.

Arnout's suggestion:

> There is a couple of thoughts that come to mind with this. I think the mission should be one level higher. In other words, what we want to achieve and not how. More along the lines of:
> 
> To change the (market) dynamics in public software development in order to decrease cost (for the tax payer), decrease risk (for the members) and increase quality for the general public.
> 
> Apart from that I don’t like the word ‘ecosystem’ and I think that if we include  open source in our mission statement it will immediately place us in a certain bucket that I don’t think we want to be in.

followed by

> **To bring transparency to public software development in order to control cost, decrease risk and improve quality.**

Reply:
I'm leaning to:

```suggestion
We help public organizations develop and maintain policy and software together.
```

Alternatively we could add another sentence:

```suggestion
We help public organizations develop and maintain policy and software together.
We do this to decrease cost, reduce risk and increase the quality of public service for everyone.
```

Reply:
@MirjamvT reminds us that we should look at the goal, not what we do.

Reply:
In our session we've just come up with the following:
```suggestion
To enable open and collaborative public-purpose software and policy.
```

We've decided not to go with a higher level mission because this is actually what we're trying to do and there are so many higher level things we would serve that this might be a very long list.